book/prover-network/usage.md

```diff
@@ -17,8 +17,8 @@ SP1_PROVER=network SP1_PRIVATE_KEY=... RUST_LOG=info cargo run --release
 
 - `SP1_PROVER` should be set to `network` when using the prover network.
 
-- `SP1_PRIVATE_KEY` should be set to your [private key](#key-setup). You will need
-  to be using a [permissioned](#get-access) key to use the network.
+- `SP1_PRIVATE_KEY` should be set to your [private key](./setup.md#key-setup). You will need
```

Comment:
These links should be `./setup` I think: https://docs.succinct.xyz/prover-network/setup

You can run mdbook locally to test, check readme

Reply:
I tested them, they're correct.